views/error-oauth.html.twig

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html.twig' %}
+{% block title %}Error - {% endblock %}
+{% block body %}
+    <div class="alert alert-danger">
+        <p>
+            <span class="glyphicon glyphicon-thumbs-down"></span>
+            <strong>An error occurred.</strong> Sorry, there was a problem logging you in to HealthPro.  Please <a href="{{ logoutUrl }}">try again</a>.
```

Comment:
Getting a twig error on this line since the `logoutUrl ` is not passed in the twig params.